src/Serilog.Settings.Configuration/Settings/Configuration/LoadedConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+using Serilog.Core;
+
+namespace Serilog.Settings.Configuration;
+
+/// <summary>
+/// Contains information about the loaded configuration.
+/// </summary>
+public class LoadedConfiguration
+{
+    internal LoadedConfiguration(IReadOnlyDictionary<string, LoggingLevelSwitch> logLevelSwitches)
+    {
+        LogLevelSwitches = logLevelSwitches;
+    }
+
+    /// <summary>
+    /// The log level switches that were loaded from the <c>LevelSwitches</c> section of the configuration.
+    /// </summary>
+    /// <remarks>The keys of the dictionary are the name of the switches, including the leading <c>$</c> character.</remarks>
```

Comment:
```suggestion
    /// <remarks>The keys of the dictionary are the names of the switches, including the leading <c>$</c> character if any.</remarks>
```

Reply:
Fixed in c58ef69e085c974e22621d233133543b2d30351a, but not the _if any_ part because the switch __always__ have a leading `$` in the `_declaredLevelSwitches` dictionary:

```csharp
public void AddLevelSwitch(string levelSwitchName, LoggingLevelSwitch levelSwitch)
{
    _declaredLevelSwitches[ToSwitchReference(levelSwitchName)] = levelSwitch;
}

string ToSwitchReference(string switchName)
{
    return switchName.StartsWith("$") ? switchName : $"${switchName}";
}
```

Reply:
Ok. I didn't know.